CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Every user-facing change should have an entry in this changelog. Please respect
 -->
 
 ## Unreleased
+- [Bugfix] Fix MongoDB replica set connection error resulting from edx-platform's pymongo (3.10.1 -> 3.12.3) upgrade ([edx-platform#30569](https://github.com/openedx/edx-platform/pull/30569)). (by @ormsbee)
```

Comment:
Can you please move that line to the CHANGELOG-nightly.md file? In the nightly branch we store changes there, otherwise there are too many conflicts when we merge the master branch.

Reply:
Done.